public/app/features/alerting/unified/home/Insights.tsx

```diff
@@ -97,7 +97,7 @@ const THIS_WEEK_TIME_RANGE = new SceneTimeRange({ from: 'now-1w', to: 'now' });
 
 const namespace = config.bootData.settings.namespace;
 
-export const INSTANCE_ID = namespace.includes('stack-') ? namespace.replace('stack-', '') : undefined;
+export const INSTANCE_ID = namespace.includes('stacks-') ? namespace.replace('stacks-', '') : undefined;
```

Comment:
I had a look and I can't see anything exposing the ID directly, wish we had that instead. I don't imagine the format would change again, but we could instead try splitting it by `-` and taking the second value if it's a number?

Reply:
I'm not sure if the word `stacks` is there for a reason...is it possible to have another namespace with `whatever-` ?

Reply:
I'm not sure, probably a good question to have an answer for anyway :D I know that on other envs its `default`. We can leave it now though of course! 🚀